### ISSUE TITLE
Confirm we can verify dependencies during trusted build process. 

### ISSUE THREAD

**Issue by yezr** [BrightSpots/rcv#756]
Part of getting certified in CA is having a trusted build process. CASOS makes a fresh build from source and then distributes that artifact to jurisdictions. 

Part of that process has been verifying the third party dependencies we use. During the 1.3.1 trusted build process each of the dependencies were verified manually. Can we brainstorm a way to make this dependency verification programmatic? 

One option is [verifying dependencies with gradle](https://docs.gradle.org/current/userguide/dependency_verification.html#:~:text=The%20verification%20metadata%20file&text=Doing%20so%2C%20Gradle%20will%20verify,%E2%80%8B). It looks like you can '[bootstrap](https://docs.gradle.org/current/userguide/dependency_verification.html#sec:bootstrapping-verification)' a set of hash checks for each dependency. After the initial bootstrapping, hash checking happens automatically when building with gradle.

**Comment by yezr** [comment]
I was able to follow the gradle instructions to bootstrap a `verification-metadata.xml` file. Github won't let me upload an `.xml` so here it is a `.txt`
[verification-metadata.txt](https://github.com/BrightSpots/rcv/files/12769123/verification-metadata.txt)

That file goes in the `_project root_\gradle\` folder. I used the gradle build action and it automatically picks it up. Original I had nothing in it but the [default text](https://mvnrepository.com/artifact/org.apache/apache/29) to turn on dependency verification. Without any checksums in the file to verify against the build failed. I used the bootstrap instructions to build the one I attached.

That bootstrap just runs the requested hash against the artifacts we get from Maven. First thing I did was head over to Maven to verify that what they have listed for hashes matches what gradle built for us. I confirmed a couple like the [apache bcel hash](https://repo1.maven.org/maven2/org/apache/bcel/bcel/6.5.0/). I can only find md5 and sha1 hashes online. The docs also suggest verifying the gradle derived hashes with the projects themselves in case the maven repo artifacts are already compromised.

If someone can point me in the right direction for correct hashes to verify against I can run through each of the artifacts and verify them.
